src/AnnouncementBubble/NormalAnnouncementBubble.less

```diff
@@ -161,9 +161,43 @@ button.NormalAnnouncementBubble--deleteMenuItem {
   }
 }
 
+.NormalAnnouncementBubble--translatedLanguages--tooltip {
+  padding-left: @size_xs;
+}
+
+.NormalAnnouncementBubble--translatedLanguages--icon {
+  .margin--right--2xs();
+  .margin--top--4xs();
+  width: 1rem;
+}
+
+.NormalAnnouncementBubble--translatedLanguages--text--desktop {
+  color: @neutral_medium_gray;
+  .text--medium;
+}
+
+.NormalAnnouncementBubble--translatedLanguages--text--mobile {
+  color: @neutral_medium_gray;
+  .text--medium;
+  display: none;
+}
+
+.NormalAnnouncementBubble--translatedLanguages--container {
+  margin-top: 1rem;
+  .flex--direction--row();
+  .flex--grow;
+
+  &:hover .NormalAnnouncementBubble--translatedLanguages--text--desktop {
+    color: @primary_blue_shade_2;
+  }
+
+  &:hover .NormalAnnouncementBubble--translatedLanguages--text--mobile {
+    color: @primary_blue_shade_2;
+  }
+}
+
 .NormalAnnouncementBubble--readReceipts--tooltip {
-  .text--semi-bold();
-  padding-left: @size_s;
+  padding-left: @size_xs;
```

Comment:
@brianbyrhee Changed a tiny bit of the read receipts CSS here for tooltip text boldness, not sure if you had it there for a specific edge case perhaps but it looks the same after I took it out as far as I can tell. LMK if you see an issue with this!

Reply:
Oh wait gimme a second it does look different now... let me check Figma

Reply:
Nvm! I was trippin